hack/components/bump-multus.sh

```diff
@@ -25,6 +25,7 @@ function __parametize_by_object() {
 				yaml-utils::update_param ${f} spec.template.metadata.labels.name 'kube-multus-ds-amd64'
 				yaml-utils::update_param ${f} spec.template.spec.containers[0].image '{{ .MultusImage }}'
 				yaml-utils::set_param ${f} spec.template.spec.containers[0].imagePullPolicy '{{ .ImagePullPolicy }}'
+				yaml-utils::set_param ${f} spec.template.spec.priorityClassName 'system-cluster-critical'
```

Comment:
like!

Reply:
Have you tried executing the bump locally to see if it really generates a valid manifets?

Reply:
sure, the 2nd changed file is made by the actual bump

Reply:
Thanks